src/util/mockDataUtils.ts

```diff
@@ -241,28 +242,45 @@ export const fakeLandingPages = (
 
 export const fakeLandingPage = (
   overrides?: Partial<LandingPage>
-): LandingPage =>
-  merge(
-    {
-      id: faker.random.uuid(),
-      buttonText: fakeLocalizedObject(),
-      buttonUrl: fakeLocalizedObject(faker.internet.url()),
-      description: fakeLocalizedObject(),
-      heroBackgroundImage: fakeLocalizedCmsImage(),
-      heroBackgroundImageColor: fakeLocalizedObject('BLACK'),
-      heroBackgroundImageMobile: fakeLocalizedCmsImage(),
-      heroTopLayerImage: fakeLocalizedCmsImage(),
-      keywords: fakeLocalizedCmsKeywords(),
-      metaInformation: fakeLocalizedObject(),
-      pageTitle: fakeLocalizedObject(),
-      socialMediaImage: fakeLocalizedCmsImage(),
-      title: fakeLocalizedObject(),
-      titleAndDescriptionColor: fakeLocalizedObject('BLACK'),
-      __typename: 'LandingPage',
-    },
-    overrides
-  );
+): LandingPage => {
+  return {
+    id: faker.random.uuid(),
+    bottomBanner: fakeBanner(),
+    buttonText: fakeLocalizedObject(),
+    buttonUrl: fakeLocalizedObject(faker.internet.url()),
+    description: fakeLocalizedObject(),
+    heroBackgroundImage: fakeLocalizedCmsImage(),
+    heroBackgroundImageColor: fakeLocalizedObject('BLACK'),
+    heroBackgroundImageMobile: fakeLocalizedCmsImage(),
+    heroTopLayerImage: fakeLocalizedCmsImage(),
+    keywords: fakeLocalizedCmsKeywords(),
+    metaInformation: fakeLocalizedObject(),
+    pageTitle: fakeLocalizedObject(),
+    socialMediaImage: fakeLocalizedCmsImage(),
+    title: fakeLocalizedObject(),
+    titleAndDescriptionColor: fakeLocalizedObject('BLACK'),
+    __typename: 'LandingPage',
+    ...overrides,
+  };
```

Comment:
lets not get rid of merge function here. it merges objects deeploy and that is why it was used here

Reply:
mixed it up with lodash assign which can be replaced by es6 spread.

Reply:
now fixed